packages/blocks/src/api/test/validation.js

```diff
@@ -662,4 +662,48 @@ describe( 'validation', () => {
 			expect( isValid ).toBe( true );
 		} );
 	} );
+
+	describe( 'validate', () => {
+		it( 'returns undefined', () => {
```

Comment:
This should be more descriptive. "Returns undefined when validation succeeds.", or something like that.